3-networks-dual-svpc/envs/production/remote.tf.cloud.example

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  default_region1  = data.tfe_outputs.bootstrap.outputs.common_config.default_region
+  default_region2  = data.tfe_outputs.bootstrap.outputs.common_config.default_region_2
+}
```

Comment:
a duplicate of this code also need to be created in the `remote.tf` file using `data.terraform_remote_state.bootstrap`

Reply:
Good catch. I had it staged locally, but forgot to include it in the last commit.
b2edb67e71b01cc80501decafb60a868708f1609

Reply:
and 4a0b103eb311ba25522cfefca59e06e777063b79